src/SWP/Bundle/CoreBundle/Controller/ContainerController.php

```diff
@@ -14,6 +14,7 @@
 
 namespace SWP\Bundle\CoreBundle\Controller;
```

Comment:
It still require `CoreBundle` and `NelmioApiBundle` and i would like to do not have those dependencies here.